themes/bootstrap3/templates/RecordDriver/SolrDefault/result-grid.phtml

```diff
@@ -9,35 +9,37 @@ $openUrlActive = $openUrl->isActive();
 $urls = $this->record($this->driver)->getLinkDetails($openUrlActive);
 ?>
 
-<div class="result <?=$this->driver->supportsAjaxStatus()?' ajaxItem':''?>">
+<div class="grid-result<?=$this->driver->supportsAjaxStatus()?' ajaxItem':''?>">
   <input type="hidden" value="<?=$this->escapeHtmlAttr($this->driver->getUniqueID())?>" class="hiddenId" />
-  <? if ((isset($this->showCartControls) && $this->showCartControls)
-      || (isset($this->showBulkOptions) && $this->showBulkOptions)): ?>
-    <?=$this->record($this->driver)->getCheckbox() ?></br>
+  <? if (isset($this->showCheckboxes) && $this->showCheckboxes): ?>
+    <label class="grid-checkbox">
+      <?=$this->record($this->driver)->getCheckbox() ?>
```

Comment:
It appears that cart checkboxes do not work in grid view. I'm seeing this problem in master as well, so I don't think it's necessarily related to this pull request. Maybe it was broken when we adjusted checkboxes a few months ago. We ought to add a test case for this!